src/timedata.cpp

```diff
@@ -98,11 +99,12 @@ void AddTimeData(const CNetAddr& ip, int64_t nOffsetSample)
         }
 
         if (LogAcceptCategory(BCLog::NET)) {
+            std::string logMessage{"time data samples: "};
```

Comment:
nit: current project style would be:

```suggestion
            std::string log_message{"time data samples: "};
```

Reply:
renamed the variable.